src/component/Nav.js

```diff
@@ -1,4 +1,4 @@
-import 'bootstrap/dist/css/bootstrap.min.css';  
+//import 'bootstrap/dist/css/bootstrap.min.css';  
```

Comment:
once this css is imported in App.css it will be accessible all over the app, no need to import it in any other component.